resources.php

```diff
@@ -38,7 +37,7 @@
 
           <ul>
           <li>
-          <a href="https://walletgenerator.net/?currency=Primecoin">Offline Paper Wallet</a></li>
```

Comment:
The paper wallet is cool and fancy, thank you, you can share it on other website, no problem. But for the official website, I just want to keep the official website official.
This is a wallet that source-unknown. So I just changed back to the official wallet. Since the official has provided a wallet, and it can generate QR code for you, why should we remove the official wallet and replace it with an unidentified wallet provided by a website origin-unknown?

Reply:
Aha Ok. My intention wasn't to remove the original wallet, I was replacing a link exactly here to and old paperwallet, there was never a link to the official wallet here. But I understand, makes sense to not 'endorse' any 3rd party wallets. 
When I finish the electrum wallet though, perhaps we can include that here? ;)

Reply:
Yes, yes.  That is exactly the reason, thanks for understanding.  We all want to make things better.
When you finish the electrum wallet, if contributors say it is nice, why not we include that?  : )

Reply:
I have redirected the wiki link to the github wiki in the meantime, until this is merged

Reply:
Hi,theapplicationist! I want to ask you a question. On the main web page header.  'Another world record broken on 30th March 2018! We found our first 15 length Cunningham Chain of the First Kind! Which has an origin with 96 digits' . I checked the Block 2,592,901, it is actually a bi-twin primes chain.  And I did not find a first 15 length Cunningham Chain of the First Kind, would you mind to tell me which block is the 15 1st kind chain or just a typo here?

Reply:
Honestly I'm not sure now. At the time I remember verifying and had some sources to back it up, but now I can't find the backup I had then, besides the mining pool reporting it (XPMForAll), but now that looks like a typo. At the time I remember finding it was a CC1/BiTwin combined and thinking it would be better to report the CC1 part.. But alas, we are where we are. 

Looks like we found a CC1 14 of 97 length today (Block 2718842) so maybe I can do an update for that one to overwrite it anyway, while I investigate further, I was going to speak to Suki at Primes.Zone. 

What do you think to asking primes.zone to provide an API (there may already be one) to keep the site records up to date? 

Is there anywhere we can talk in realtime btw? Telegram / Primecoin chat?

Reply:
Oh, thank you. 

I like the idea to use an API provided by prime.zone.

Reply:
Great, I've managed to obtain an API we can use, so I'll put something together for that later on. I'll tag you in the PR for review.